neovim/msgpack_rpc/event_loop/uv.py

```diff
@@ -73,12 +73,11 @@ def _connect_child(self, argv):
                             flags=pyuv.UV_CREATE_PIPE + pyuv.UV_WRITABLE_PIPE)
         stderr = pyuv.StdIO(self._error_stream,
                             flags=pyuv.UV_CREATE_PIPE + pyuv.UV_WRITABLE_PIPE)
-        self._process = pyuv.Process(self._loop)
-        self._process.spawn(file=argv[0],
-                            exit_callback=self._on_exit,
-                            args=argv[1:],
-                            flags=pyuv.UV_PROCESS_WINDOWS_HIDE,
-                            stdio=(stdin, stdout, stderr,))
+        pyuv.Process.spawn(self._loop,
+                           args=argv,
+                           exit_callback=self._on_exit,
+                           flags=pyuv.UV_PROCESS_WINDOWS_HIDE,
+                           stdio=(stdin, stdout, stderr,))
```

Comment:
Well, we don't need to do it in this PR, but I noticed that [here](https://github.com/meiralins/python-client/blob/c165783367703d8a61b9a0f2b32f20c244ef567b/neovim/msgpack_rpc/event_loop/__init__.py#L8) we choose asyncio or pyuv. So as part of that choice, check the API as described in my above comment: if `pyuv.Process.spawn` is _not_ an instance method, then we known we have pyuv >= 1.x, and we [choose](https://github.com/meiralins/python-client/blob/c165783367703d8a61b9a0f2b32f20c244ef567b/neovim/msgpack_rpc/event_loop/__init__.py#L8) `EventLoop = UvEventLoop`. Else, choose `EventLoop = AsyncioEventLoop`